project/AppDependencies.scala

```diff
@@ -6,33 +6,35 @@ object AppDependencies {
 
   val compile = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo"         %% "play2-reactivemongo"             % "0.20.11-play27",
+    "org.reactivemongo"         %% "play2-reactivemongo"             % "0.20.11-play28",
     "org.reactivemongo"         %% "reactivemongo-bson-api"          % "0.20.11",
     "com.typesafe.play"         %% "play-iteratees"                  % "2.6.1",
     "com.typesafe.play"         %% "play-iteratees-reactive-streams" % "2.6.1",
```

Comment:
Leaving a note here regarding the `play-iteratees` package, maybe this needs to be looked into in a separate issue:
> https://github.com/playframework/play-iteratees
> End of Life
> The active Playframework contributors consider this repository has reached End of Life and archived it.
> This repository is not being used anymore and won't get any further updates.